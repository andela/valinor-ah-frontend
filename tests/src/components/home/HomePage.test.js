import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import Home from '../../../../src/components/home/HomePage';

test('Home snapshot test', () => {
  const component = renderer.create(<Router><Home /></Router>);
  const tree = component.toJSON();
  // eslint-disable-next-line no-undef
  expect(tree).toMatchSnapshot();
});
