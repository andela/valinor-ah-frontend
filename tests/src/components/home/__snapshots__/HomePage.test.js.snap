// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot test 1`] = `
<div>
  <header>
    <nav
      className="navbar navbar-expand-lg navbar-light bg-light"
    >
      <a
        className="navbar-brand logo"
        href="/"
      >
        Author's Haven |
      </a>
      <a
        aria-current="page"
        className="nav-item nav-link active active"
        href="/"
        onClick={[Function]}
        style={
          Object {
            "color": "red",
          }
        }
      >
        Home
      </a>
      <a
        aria-current={null}
        className="nav-item nav-link"
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </nav>
  </header>
  <h1>
    Home page here....
  </h1>
</div>
`;
